README.md

```diff
@@ -105,7 +105,7 @@ it because everything will have to rebuild.
 ### Connect to the containers
 
 #### react
-`docker exec -it react /bin/bash`
+`docker exec -it react /bin/sh`
```

Comment:
Noticed I don't have bash installed on the react container right now so the command `docker exec -it react /bin/bash` doesn't work.